addons/account/views/product_view.xml

```diff
@@ -9,16 +9,18 @@
             <field name="arch" type="xml">
                 <page string="Sales" position="after">
                     <page string="Accounting" groups="account.group_account_invoice">
-                        <group>
-                            <label for="categ_id" string="Internal Category"/>
-                            <div><field name="categ_id" colspan="3" nolabel="1"/></div>
+                        <group name="product_category">
+                            <group name="category">
+                                <label for="categ_id" string="Internal Category"/>
+                                <div><field name="categ_id" colspan="3" nolabel="1"/></div>
+                            </group>
                         </group>
                         <group name="properties">
-                            <group>
+                            <group name="sale_tax_part">
```

Comment:
names are very useful when inheriting the views, it is really appreciated to have a name on which we can rely (rather than strings which are expected to change sometimes)